frontend/mulighetsrommet-veileder-flate/src/components/modal/Delemodal.tsx

```diff
@@ -149,7 +153,7 @@ const Delemodal = ({ modalOpen, setModalOpen, tiltaksgjennomforingsnavn, brukerN
             minRows={10}
             maxRows={50}
             data-testid="textarea_tilbakemelding"
-            maxLength={1000}
+            maxLength={MAKS_ANTALL_TEGN}
```

Comment:
Hvis vi setter en maksbegrensning her så burde vi kanskje også ha en tilsvarende begrensning på malen i Sanity også?

Reply:
Ja, Signe også om å gjøre følgende: `maks antall tegn = malen fra sanity + 200 tegn`. Jeg tror jeg går for det i første omgang, siden det er såpass uklart hvordan det blir med tekstene i Sanity foreløpig